scripts/pre-commit.sh

```diff
@@ -12,4 +13,28 @@ if [ $EXIT_CODE -ne 0 ]; then
   echo "***********************************************"
   exit $EXIT_CODE
 fi
-rm $OUTPUT
\ No newline at end of file
+rm $OUTPUT
+
+echo "Running metalava (defaults) check..."
+./gradlew metalavaCheckCompatibilityDefaultsRelease -q
+EXIT_CODE=$?
+if [ $EXIT_CODE -ne 0 ]; then
+  echo "❌ metalava for defaults flavor failed, running it for you..."
+
+  ./gradlew metalavaCheckCompatibilityDefaultsRelease -q
+
+  echo "API dump done, please check the results and then try your commit again!"
```

Comment:
Maybe we should mention that the updated dump should be included in the commit? Something like:
```suggestion
  echo "API dump done. Please check the updated API dump and add it to your commit. Then try again!"
```

(Same on line 38 below.)

Reply:
This sounds much clearer. Thank you for your suggestion!